app/services/labor/ping_admins.rb

```diff
@@ -0,0 +1,26 @@
+module Labor
+  class PingAdmins
+    def initialize(user)
+      @user = user
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      return unless user && Rails.env.production?
```

Comment:
It would be nice to add specs for this class, but it's not easy because it has a `Rails.env.production?` check. Probably, it should be a topic for another pr, I'll look into this later.

Reply:
@lightalloy I thought the same.